buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/fixtures/TestFixturesPlugin.kt

```diff
@@ -109,32 +130,18 @@ open class TestFixturesPlugin : Plugin<Project> {
                 }
             }
         }
-
-        plugins.withType<EclipsePlugin> {
```

Comment:
So eclipse should work out of the box now?

Reply:
yes